tox.ini

```diff
@@ -11,24 +11,32 @@ deps =
   mypy: mypy~=0.711
 
 setenv =
-  PYTHONPATH={toxinidir}/opentelemetry-api/src/
-  mypy: MYPYPATH={env:PYTHONPATH}
+  mypy: MYPYPATH={toxinidir}/opentelemetry-api/src/
 
 changedir =
-  test: opentelemetry-api/tests
+  test-api: opentelemetry-api/tests
+  test-sdk: opentelemetry-sdk/tests
```

Comment:
We may not want separate test targets in the long run, but this was the easiest way to get test discovery to work since `opentelemetry` isn't a package (with an `__init__.py`) anymore.

Reply:
We want to remove `opentelemetry/__init__.py` after we workaround the pylint discovery issue https://github.com/open-telemetry/opentelemetry-python/pull/37#issuecomment-506570212.